### ISSUE TITLE
Card counting used in decision making

### ISSUE THREAD

**Issue by ulissesferreira** [UlissesFerreira/BlackjackSimulation#1]
Due to the fact that the casino keeps the decks shuffled, the card count needs to be reset every hand. Unfortunately for the player, the casino uses 4 to 6 decks, which lowers the importance of each card on the True Count. For simplicity's sake, the player doesn't use the True Count obtained to make his decisions. It would be an interesting experiment to include the complete count presented by Edward O. Thorp to see how much of a match the player would be to the odds. 

I predict that even with those changes implemented, the Casino would still have the upper hand, due to the aforementioned fact. Maybe with fewer decks, those probabilities would change.

TL: DR - Change the decision-making engine to take into account the True Count of the cards.
